### ISSUE TITLE
[Bug report] Second hand disabled and crashing when accessed

### ISSUE THREAD

**Issue by cbrueffer** [Pyrdacor/Ambermoon.net#283]
**Describe the bug**
See screenshot, the second hand of the main character is disabled. The game crashes when placing an item (e.g. shield) in the second hand, or bi-handed weapon into the first hand.

```
Unhandled exception. System.Collections.Generic.KeyNotFoundException: The given key '0' was not present in the dictionary.
   at System.Collections.Generic.Dictionary`2.get_Item(TKey key)
   at Ambermoon.UI.ItemGrid.DropItem(Int32 slot, DraggedItem item)
   at Ambermoon.UI.ItemGrid.Click(Position position, DraggedItem draggedItem, ItemAction& itemAction, MouseButtons mouseButtons, CursorType& cursorType, Action`1 dragHandler)
   at Ambermoon.UI.Layout.Click(Position position, MouseButtons buttons, CursorType& cursorType, UInt32 currentTicks, Boolean pickingNewLeader, Boolean pickingTargetPlayer, Boolean pickingTargetInventory)
   at Ambermoon.Game.OnMouseDown(Position position, MouseButtons buttons)
   at Ambermoon.GameWindow.Mouse_MouseDown(IMouse mouse, MouseButton button)
   at Silk.NET.Input.Internals.MouseImplementationBase.HandleMouseDown(IMouse mouse, MouseButton button)
   at Silk.NET.GLFW.Glfw.PollEvents()
   at Silk.NET.Windowing.WindowExtensions.<>c__DisplayClass2_0.<Run>b__0()
   at Silk.NET.Windowing.Internals.ViewImplementationBase.Run(Action onFrame)
   at Silk.NET.Windowing.WindowExtensions.Run(IView view)
   at Ambermoon.GameWindow.Run(Configuration configuration)
Fatal error. Internal CLR error. (0x80131506)
   at Silk.NET.GLFW.Glfw.PollEvents()
   at Silk.NET.Windowing.WindowExtensions+<>c__DisplayClass2_0.<Run>b__0()
   at Silk.NET.Windowing.Internals.ViewImplementationBase.Run(System.Action)
   at Silk.NET.Windowing.WindowExtensions.Run(Silk.NET.Windowing.IView)
   at Ambermoon.GameWindow.Run(Ambermoon.Configuration)
   at Ambermoon.Program.Main()
Aborted (core dumped)
```

**To Reproduce**
I'm not sure what lead to the situation. The last thing I remember doing was placing a crowbar as a weapon and fighting with that.

**Expected behavior**
Both hands are functional without crashing the game.

**Screenshots**

![Screenshot_2021-12-23_11-20-01](https://user-images.githubusercontent.com/2705287/147226361-20e0ae90-4f9e-4360-b8ce-c50de881e5c7.png)

**Desktop (please complete the following information):**
 - OS: Linux
 - Version 1.3

**Comment by Pyrdacor** [comment]
Thanks for reporting. This is most likely a duplicate of #212 and #238. This crash is very hard to reproduce. This is why it isn't fixed yet.

If you have any more information what happened in the fight or what you have done prior to it, this would be very valuable.

**Comment by Pyrdacor** [comment]
Closed because it's a duplicate of #238 and tracked there.
